problemsets-mat223/linearalbegra.tex

```diff
@@ -2179,9 +2179,36 @@ \section*{Projections}
 	\begin{parts}
 		\item Make a sketch with $\vec c$, $K$, and $\Proj_K\vec c$ (you don't need to compute 
 		$\Proj_K\vec c$ exactly).
+			\begin{solution}
+				XXXX
+			\end{solution}
 		\item What should $(\vec c-\Proj_K\vec c)\cdot \mat{1\\2}$ be? Explain.
+			\begin{solution}
+				It should equal zero. 
+
+				From our picture we can see that $c-\Proj_K\vec c$ is perpendicular
+				to the line $K$, and so the dot product of this vector with any 
+				direction vector for $K$ should be zero.
+			\end{solution}
 		\item Use your formula from the previous part to find $\Proj_K\vec c$
 			\emph{without} computing any distances.
+			\begin{solution}
+				$\Proj_K\vec c = \frac{1}{5}\mat{11\\12}$
+
+				If $\Proj_K\vec c = \mat{x\\y}$, the previous part tells us
+				\[
+					\left(\mat{1\\3}-\mat{x\\y}\right)\cdot \mat{1\\2}
+					=1-x+6-2y=0
+					\quad \iff \quad
```

Comment:
Don't abbreviate `iff`. Students might not know it.

Reply:
`\iff` is the LaTeX command for the double sided implication. It spaces itself better than `\Leftrightarrow`. 

Do you want me to write out "if and only if" in there?